dependencyManagement/build.gradle.kts

```diff
@@ -96,7 +96,7 @@ val DEPENDENCIES = listOf(
   "commons-logging:commons-logging:1.3.0",
   "commons-validator:commons-validator:1.8.0",
   "io.netty:netty:3.10.6.Final",
-  "io.opentelemetry.contrib:opentelemetry-aws-xray-propagator:1.28.0-alpha",
+  "io.opentelemetry.contrib:opentelemetry-aws-xray-propagator:1.34.0-alpha-SNAPSHOT",
```

Comment:
looks like we need to remove a snapshot repo reference somewhere...

Reply:
should be fixed with https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/10761